src/viam_mlmodelservice_triton_impl.cpp

```diff
@@ -103,6 +103,12 @@ class Service : public vsdk::MLModelService, public vsdk::Stoppable, public vsdk
             std::lock_guard<std::mutex> lock(state_lock_);
             if (!stopped_) {
                 stopped_ = true;
+
+                // Remove any symlinks we've added to our module workspace.
+                if (!state_->model_name.empty()) {
+                    remove_symlink_mlmodel_(state_->model_name);
```

Comment:
Could responsibility for removing the symlink be made part of `State` destruction?